### DISCUSSION TITLE
Is there an integrated terminal in the Edwood editor?

### DISCUSSION THREAD

**Discussion by plamendraganov** [rjkroege/edwood#505] [Q&A]
I am running Edwood on Windows 11. I know that clicking with button 2 on 'win' in Acme-sac is opening a terminal where I can write some shell commands. However, it seems that this is not working on Edwood. 
Is there a way to open a terminal within Edwood?

**Comment by rjkroege** [comment]
I think that Paul said that `cmd/win` runs on Windows. So if you compile that `win`, it should work? (I've not tried it on windows though so maybe you can try it and comment?)

**Reply by VitGinz** [reply]
@rjkroege 
I try to run win.exe, it doesn't work, prints ""failed to open acme window".
Do you have any tips?

**Reply by rjkroege** [reply]
Not really. Sorry. I tried setting `NAMESPACE` by hand based on what `edwood.exe` prints as the address. But I think `win.exe` is being stupid about where to look for Edwood's implementation of the Acme namespace. I'd welcome patches, additional details on what you've tried etc. Otherwise, maybe create a new bug for "fix `win` on Windows" or something like that and I'll see about fixing it.
